app/uk/gov/hmrc/helptosavefrontend/config/CustomWSConfigParser.scala

```diff
@@ -144,7 +144,7 @@ class CustomWSConfigParser @Inject()(configuration: Configuration, env: Environm
       .map(password ⇒ Base64.getDecoder.decode(password))
       .map(bytes ⇒ new String(bytes))
 
-    ks.copy(data = None, filePath = Some(ksFilePath), password = decryptedPass)
+    ks.copy(data     = None, filePath = Some(ksFilePath), password = decryptedPass)
```

Comment:
scalariform aligns the first equals sign here with the equals sign on line 143 - don't know why!